model_analyzer/perf_analyzer/perf_analyzer.py

```diff
@@ -323,14 +323,33 @@ def _get_cmd(self):
         return cmd
 
     def _get_single_model_cmd(self, index):
-        cmd = [self.bin_path]
-        if self._is_multi_model():
-            cmd += ["--enable-mpi"]
-        cmd += self._get_pa_cli_command(index).replace("=", " ").split()
+        if self._model_type == "LLM":
+            cmd = [
+                "genai-perf",
+                "-m",
+                self._config.models_name(),
+                "--streaming",
```

Comment:
not sure that we want to hardcode streaming. Should at least create a story to hook this up to something, either the online vs offline mode or some new LLM flag

Reply:
I have a story to track this change as part of adding all the genai-perf CLI options.